From e9a8272aa254f1e0b15781910167f10c112bf320 Mon Sep 17 00:00:00 2001
From: Tianling Shen <cnsztl@immortalwrt.org>
Date: Sun, 4 Jan 2026 20:10:02 +0800
Subject: [PATCH] Revert "exim: adjust with glibc and libcrypt-compat"

This commit only applies to master and 25.12 branch which have newer
glibc (2.39). We don't have `libcrypt-compat` package on 24.10 branch.

This reverts commit 80c90e9049092b0f709b7e534f8210d42ec4a92e.

Signed-off-by: Tianling Shen <cnsztl@immortalwrt.org>
---
 mail/exim/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mail/exim/Makefile b/mail/exim/Makefile
index 5e8bd3687758e..2c3a450ff2496 100644
--- a/mail/exim/Makefile
+++ b/mail/exim/Makefile
@@ -45,7 +45,7 @@ LOOKUPS:= \
 define Package/exim/Default
   SECTION:=mail
   CATEGORY:=Mail
-  DEPENDS:= +USE_GLIBC:libcrypt-compat +libdb47 +libpcre2 $(ICONV_DEPENDS) +BUILD_NLS:libidn2 +BUILD_NLS:libidn
+  DEPENDS:=+libdb47 +libpcre2 $(ICONV_DEPENDS) +BUILD_NLS:libidn2 +BUILD_NLS:libidn
   TITLE:=Exim message transfer agent
   URL:=http://www.exim.org/
   USERID:=exim=42:exim=42
